rotkehlchen/tests/unit/decoders/test_aave.py

```diff
@@ -216,3 +223,668 @@ def test_aave_eth_withdraw_v1(database, ethereum_inquirer):
+                topics=[
+                    hexstring_to_bytes('0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000030ba81f1c18d280636f32af80b9aad02cf0854e'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000006b44ba0a126a2a1a8aa6cd1adeed002e141bcd44'),  # noqa: E501
+                ],
+            ), EvmTxReceiptLog(
+                log_index=26,
+                data=hexstring_to_bytes('0x00000000000000000000000000000000000000000000012a66d9c49e79440554'),  # noqa: E501
+                address=string_to_evm_address('0x030bA81f1c18d280636F32af80b9AAd02Cf0854e'),
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000002715273613632226985186221669179813245119'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000000000000000000000000000000000000000000000'),  # noqa: E501
+                ],
+            ), EvmTxReceiptLog(
+                log_index=28,
+                data=hexstring_to_bytes('0x00000000000000000000000000000000000000000000012a66d9c49e79440554'),  # noqa: E501
+                address=string_to_evm_address('0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9'),
+                removed=False,
+                topics=[
+                    hexstring_to_bytes('0x3115d1449a7b732c986cba18244e897a450f61e1bb8d589cd2e69e6c8924f9f7'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000002715273613632226985186221669179813245119'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000006b44ba0a126a2a1a8aa6cd1adeed002e141bcd44'),  # noqa: E501
+                ],
+            ),
+        ],
+    )
+    dbevmtx = DBEvmTx(database)
+    with dbevmtx.db.user_write() as cursor:
+        dbevmtx.add_evm_transactions(cursor, [transaction], relevant_address=None)
+    decoder = EthereumTransactionDecoder(
+        database=database,
+        ethereum_inquirer=ethereum_inquirer,
+        transactions=eth_transactions,
+    )
+    with dbevmtx.db.user_write() as cursor:
+        events = decoder.decode_transaction(cursor, transaction=transaction, tx_receipt=receipt)
+    expected_events = [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=ZERO, usd_value=ZERO),
+            location_label='0x2715273613632226985186221669179813245119',
```

Comment:
here and in the other tests you already have a variable for this, `user_address`. Use that one